src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -166,6 +166,7 @@ public void ApplyShowPreviewSetting(bool show)
             UserSettings settings,
             Configuration.ISettings nugetSettings)
         {
+            var dependencySetting = GetDependencyBehaviorFromConfig(nugetSettings);
```

Comment:
nugetSettings can be null, which will throw in GetDependencyBehaviorFromConfig. Move this down to just above SetSelectedDepencyBehavior